geonames/cities.py

```diff
@@ -0,0 +1,31 @@
+from .models import GeoName
+
+
+def get_city_coords(city=None, state=None, iso3='USA'):
+    """Returns a tuple of (latitude, longitude) values for the given city
+    and state.
+
+    Returns (None, None) if the city or state name cannot be located.
+    If there is more than one match, the first is returned.  The
+    country searched is the United States, by default, but that can be
+    changed by passing an ISO 3166 country code as the third argument.
+
+    See https://unstats.un.org/unsd/tradekb/Knowledgebase/Country-Code
+    for more information.
+
+    """
+    city_coords = (None, None)
+
+    if not city or not state:
+        return city_coords
+
+    latlong = GeoName.objects.filter(
+        name=city,
+        regcode=state,
+        isocode__iso3=iso3
+    ).values_list('latitude', 'longitude')
```

Comment:
I wonder if case insensitive filtering makes more sense with city names since the city names in incident are just text fields, and the cases might differ from the json? 🤔

Reply:
This is a good idea, too. Not everyone uses the same case for the city names. There might be some other differences but at least this one is easy to account for.

The data set I downloaded actually does include a set of "alternate names" for each GeoName, but it seemed like it might be a fair amount of work to incorporate those into the lookup for not all that much gain. But I could be wrong about that!

Reply:
Yeah I was thinking as well that there might be other differences, but case seems easiest to address.